needles/anaconda/help/rocky-anaconda_help_localization_link-20250529.json

```diff
@@ -0,0 +1,17 @@
+{
+  "area": [
+    {
+      "height": 21,
+      "type": "match",
+      "width": 449,
+      "ypos": 366,
+      "xpos": 93
+    }
+  ],
+  "properties": [],
+  "tags": [
+    "ENV-DISTRI-rocky",
+    "ENV-LANGUAGE-english",

```

Comment:
I'm not sure we do now or will ever test anaconda_help in an alternate language. I would suggest we remove ENV-LANGUAGE-english from these needles or, if we want to ensure this IS tested in alternate languages add this tag to all English needles in preparation for adding needles for alt languages.